animareflection/ui
# Style context provider

## Description

##### Task link: https://trello.com/c/j6ZoOCMf/320-migrate-to-use-style-context-provider-for-primitives

- Migrated to [style context provider](https://panda-css.com/docs/concepts/slot-recipes#styling-jsx-compound-components) for primitive/multipart components
- Swept through components and made patches for broken styles

## Test Steps

- Verify explicit `className` instances are removed from prebuilt components
- Verify primitive "base" (root) multipart components are wrapped in `withProvider`
- Verify primitive "non-base" multipart components are wrapped in `withContext`
- Verify components render correctly. Tests check that they render *at all*, but be sure to check if they look good still (this step verifies that slots/provider wrappings weren't missed anywhere)
